### PR TITLE
[hi] Localize content/en/virtual-machine.md

### LINKED ISSUE #1237: [hi] Localize term virtual-machine into Hindi
localization of virtual-machine.md into Hindi

- https://glossary.cncf.io/virtual-machine/

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>
Fixes #1237

### CODE REVIEW

**Comment by taniaduggal** [comment]
@jayesh-srivastava

**Comment by taniaduggal** [comment]
@anubha-v-ardhan

File: content/hi/virtual-machine.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: वर्चुअल मशीन (Virtual Machine)
+status: Completed
+category: टेक्नोलॉजी 
+---
+
+### यह क्या है
+
+एक वर्चुअल मशीन (VM) एक कंप्यूटर और उसका ऑपरेटिंग सिस्टम है जो विशिष्ट हार्डवेयर से नहीं जुड़ा होता है। VMs को एक भौतिक कंप्यूटर को कई वर्चुअल मशीनों में विभाजित करने के लिए [वर्चुअलाइजेशन](/virtualization/) की आवश्यकता होती है। यह विभाजन संगठनों(organizations) और बुनियादी ढांचा प्रदाताओं(infrasteructure providers) को हार्डवेयर को प्रभावित किए बिना आसानी से VM बनाने और हटाने की अनुमति देता है।
```
Comment (Garima-Negi) [inline content/hi/virtual-machine.md:9]: Theres a typo in `infrastructure`

Comment (Garima-Negi) [reply content/hi/virtual-machine.md:9]: rest all LGTM!

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Review by Garima-Negi — APPROVED** [review]

**Comment by seokho-son** [comment]
Whereas there are some options to resolve the failure on the Pull Request Labeler action,
I am going to force merge this PR, considering #1582 :)

**Review by seokho-son — APPROVED** [review]

**Pull request merged**